include/tachimawari/control/controller/packet/protocol_1/status/bulk_read_data.hpp

```diff
@@ -36,33 +36,27 @@ namespace tachimawari::control::protocol_1
 class BulkReadData : public Packet
 {
 public:
-  enum
-  {
-    MAX_LENGTH = 255
-  };
+  enum { MAX_LENGTH = 255 };
 
   static void insert_all(
-    std::shared_ptr<std::map<uint8_t, BulkReadData>> bulk_data,
+    std::shared_ptr<std::map<uint8_t, BulkReadData>> & bulk_data,
     const BulkReadPacket & bulk_read_packet);
 
-  static int validate(
-    std::shared_ptr<std::vector<uint8_t>> rxpacket,
-    int packet_length);
+  static int validate(const std::shared_ptr<std::vector<uint8_t>> & rxpacket, int packet_length);
 
   static int update_all(
-    std::shared_ptr<std::map<uint8_t, BulkReadData>> bulk_data,
-    std::vector<uint8_t> rxpacket, int packet_length, int data_number);
+    std::shared_ptr<std::map<uint8_t, BulkReadData>> & bulk_data, std::vector<uint8_t> rxpacket,
```

Comment:
Is it possible to use const ref on `rxpacket`?

```suggestion
    std::shared_ptr<std::map<uint8_t, BulkReadData>> & bulk_data, const std::vector<uint8_t> & rxpacket,
```